plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -936,8 +935,30 @@ public void testIdentityColumns()
      */
     @Test
     public void testDeletionVectors()
+            throws Exception
     {
-        assertQuery("SELECT * FROM deletion_vectors", "VALUES (1, 11)");
+        String tableName = "deletion_vectors" + randomNameSuffix();
+
+        Path tableLocation = Files.createTempFile(tableName, null);
+        copyDirectoryContents(new File(Resources.getResource("databricks122/deletion_vectors").toURI()).toPath(), tableLocation);
+        assertUpdate("CALL system.register_table('%s', '%s', '%s')".formatted(getSession().getSchema().orElseThrow(), tableName, tableLocation.toUri()));
+
+        assertQuery("SELECT * FROM " + tableName, "VALUES (1, 11)");
+
+        assertUpdate("INSERT INTO " + tableName + " VALUES (3, 31), (3, 32)", 2);
+        assertQuery("SELECT * FROM " + tableName, "VALUES (1, 11), (3, 31), (3, 32)");
+
+        assertUpdate("DELETE FROM " + tableName + " WHERE a = 3 AND b = 31", 1);
+        assertQuery("SELECT * FROM " + tableName, "VALUES (1, 11), (3, 32)");
+
+        assertUpdate("UPDATE " + tableName + " SET a = -3 WHERE b = 32", 1);
+        assertQuery("SELECT * FROM " + tableName, "VALUES (1, 11), (-3, 32)");
+
+        assertUpdate("MERGE INTO " + tableName + " t USING " + tableName + " s " +
+                "ON (t.a = s.a) WHEN MATCHED THEN UPDATE SET b = -1", 2);
+        assertQuery("SELECT * FROM " + tableName, "VALUES (1, -1), (-3, -1)");
```

Comment:
Reading logic for deletion vectors might be broken though I'm not sure if Databricks writes such transaction logs:
```
java.lang.AssertionError: For query 20240501_092829_00018_hvqds: 
 SELECT * FROM deletion_vectorsg763ca8zs4
not equal
Actual rows (up to 100 of 1 extra rows shown, 3 rows in total):
    [2, 22]
Expected rows (up to 100 of 0 missing rows shown, 2 rows in total):
```